### PR TITLE
update to ledger-client@0.8.77

### PR DESCRIPTION
**Pull request by mrose17**
Auditor: @ayumi 

in preparation to resolve https://github.com/brave/browser-laptop/issues/3350 (by using `client.recoverWallet`) and https://github.com/brave/browser-laptop/issues/4058 (by using `client.setTimeUntilReconcile`).

### CODE REVIEW

**Comment by ayumi** [comment]
Minor: For future PRs with package upgrades, can you summarize changes? it will help expedite reviewing :)

**Comment by mrose17** [comment]
sure. 
- ledger-client@0.8.77:
 - add setTimeUntilReconcile method to be used on https://github.com/brave/browser-laptop/issues/4058
 - add recoverWallet to be used on https://github.com/brave/browser-laptop/issues/3350

- ledger-publisher@0.8.69:
 - just the latest version with fixes from running `npm run lint`

**Comment by ayumi** [comment]
Reviewed the referenced packages.

Noted `ledger-publisher` was also updated; may want also to reference that version bump in the PR/commit.

lgtm :jack_o_lantern:

**Comment by mrose17** [comment]
many thanks!
